pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -889,11 +891,13 @@ func (p *MasqueradePodInterface) createBridge() error {
 		return err
 	}
 
+	mac, _ := net.ParseMAC(hardcodedMasqueradeMAC)
 	// Create a bridge
 	bridge := &netlink.Bridge{
 		LinkAttrs: netlink.LinkAttrs{
-			Name: p.bridgeInterfaceName,
-			MTU:  int(p.vif.Mtu),
+			Name:         p.bridgeInterfaceName,
```

Comment:
According to https://github.com/kubevirt/kubevirt/pull/4581/commits/6c82c03269bf5d701ced8ee3b494ccbc6f3c6bc9

> When you create a bridge device it initially has a MAC address of all zeros,
> and when you add NIC devices to the bridge, its MAC address gets update to
> the numerically lowest MAC address of all the NICs.

Cannot the MAC you set here on the bridge be overridden by the tap or the eth0?
If yes, I would solve this problem by setting the MAC on the tap and not on the bridge. Giving the tap a low address (maybe starting with 00?)

Reply:
> With this change, I think you can remove the `Eventually` from the [migration test](https://github.com/kubevirt/kubevirt/blob/master/tests/vmi_networking_test.go#L784).
> IIRC, the `Eventually` is there since it takes the vm ARP table time to be refreshed with the new mac of the default route (k6t-eth0 bridge).

Correct; gone.

> Cannot the MAC you set here on the bridge be overridden by the tap or the eth0?
If yes, I would solve this problem by setting the MAC on the tap and not on the bridge. Giving the tap a low address (maybe starting with 00?)

Yup, defined it to the lowest possible local unicast address - `02:00:00:00:00:00`.